test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -231,9 +266,13 @@ export default class PerfCompare extends Component {
             return
         }
 
-        let baselineRunJSON = new JenkinsRunJSON(resBenchmarkBaselineJson);
-        let testRunJSON = new JenkinsRunJSON(resBenchmarkTestJson);
-        
+        let baselineRunJSON = new ExtractRelevantJenkinsTestResults(resBenchmarkBaselineJson);
+        let testRunJSON = new ExtractRelevantJenkinsTestResults(resBenchmarkTestJson);
+        await this.getJenkinsRawData(resBenchmarkBaselineJson);
+        let childBuildsListBaseline = this.state.jenkinsRawData;
```

Comment:
We could directly return the data and put it in `childBuildsListBaseline` instead of using `this.state.jenkinsRawData`.

Reply:
`childBuildsListBaseline` isn't very clear what's it's storing. We could rename it to `childrenBuildRawData`.